query-connector/src/app/ui/designSystem/table/Table.tsx

```diff
@@ -15,26 +17,27 @@ type TableProps = {
  * @param root0.bordered - whether to render a bordered table
  * @param root0.striped - whether to render a striped table
  * @param root0.className - additional custom class names
+ * @param root0.fullWidth - whether to render with fullWidth mode
  * @returns - A UWSDS-styled table
  */
 const Table: React.FC<TableProps> = ({
   children,
-  bordered = true,
+  bordered = false,
```

Comment:
Think we'll need to swap the tables in `query-connector/src/app/(pages)/query/components/resultsView/tableComponents` to use the unbordered version.

Before:
<img width="1360" alt="Screenshot 2025-02-06 at 4 32 16 PM" src="https://github.com/user-attachments/assets/85b7331d-7d66-4411-8fbe-9f1f48274564" />

After: 
<img width="1019" alt="Screenshot 2025-02-06 at 4 32 33 PM" src="https://github.com/user-attachments/assets/d9e1d679-293f-45d8-9d48-2f057d6ce2d5" />